app/services/csv/lettings_log_csv_service.rb

```diff
@@ -51,7 +51,13 @@ def set_csv_attributes
 
     def ordered_form_questions
       downloaded_form_years = LettingsLog.all.map(&:collection_start_year).uniq.compact
-      downloaded_form_fields = downloaded_form_years.count == 1 && downloaded_form_years[0].present? ? FormHandler.instance.get_form("#{downloaded_form_years[0]}_#{downloaded_form_years[0] + 1}").questions : FormHandler.instance.forms.first.second.questions
+
+      if downloaded_form_years.count == 1 && downloaded_form_years[0].present?
```

Comment:
It's not super clear what's going on here, but I can see you haven't changed it materially in this PR, so let's come back to it!

Reply:
I think the purpose of this part is to order the columns in csv file in the same order as the form questions in the UI.
This if statement is choosing which form to use to get the questions order. If the csv is for records from only one year (for example, the list was filtered down to one year) it will choose the form for that year. Otherwise it will choose a default form. It's not the clearest, so refactoring this might be a good idea